pkg/acsengine/k8s_versions.go

```diff
@@ -8,6 +8,32 @@ import (
 )
 
 var k8sComponentVersions = map[string]map[string]string{
+	"1.10": {
```

Comment:
@feiskyer Can you comment on the image references below for v1.10? Especially if there are additional k8s componentry (manifests/addons) that we want to build into the foundations for these clusters.

Reply:
@jackfrancis The versions are still not clear now. We may need to wait for the formal 1.10 release.